src/utils/mapUtils.ts

```diff
@@ -6,3 +8,16 @@ export function computeIfAbsent(map: Map<unknown, unknown>, key: unknown, defaul
     }
     return value;
 }
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function computeIfAbsentCache(cache: LRUCache<any, any>, key: unknown, defaultValueFn: () => unknown) {
```

Comment:
I'd suggest to rename this to something like `computeIfUncached`. Makes its purpose a bit clearer, imo